src/simulation/thermal/sensorThermal/sensorThermal.h

```diff
@@ -57,31 +67,44 @@ class SensorThermal: public SysModel {
     Message<TemperatureMsgPayload> temperatureOutMsg; //!< output temperature message
 
     Eigen::Vector3d nHat_B;                     //!< [-] Sensor normal unit vector relative to the spacecraft body frame.
+    TempFaultState_t faultState;                 //!< [-] Fault status variable
 
     double sensorPowerDraw;                     //!< [W] Power consumed by the sensor (+).
-    uint64_t sensorStatus;                        //!< [-] Sensor status (0/1)
+    uint64_t sensorStatus;                      //!< [-] Sensor status (0/1)
     double sensorArea;                          //!< [m^2] Sensor area in meters squared
     double sensorAbsorptivity;                  //!< [-] Sensor absorptivity (between 0 and 1)
     double sensorEmissivity;                    //!< [-] Sensor emissivity (between 0 and 1)
     double sensorMass;                          //!< [kg] Sensor mass in kg
     double sensorSpecificHeat;                  //!< [J/kg/K] Sensor specific heat
     double T_0;                                 //!< [C] Initial temperature
 
-    BSKLogger bskLogger;                          //!< -- BSK Logging
+    double senBias;                             //!< [-] Sensor bias value
+    double senNoiseStd;                         //!< [-] Sensor noise value
+    double walkBounds;                          //!< [-] Gauss Markov walk bounds
+    double pastValue;                           //!< [-] measurement from last update (used only for faults)
+    double stuckValue;                          //!< [C] Value for temp sensor to get stuck at
+    double spikeProbability;                    //!< [-] Probability of spiking at each time step (between 0 and 1)
+    double spikeAmount;                         //!< [-] Spike multiplier
+
+
+    BSKLogger bskLogger;                         //!< -- BSK Logging
 
 private:
     TemperatureMsgPayload temperatureMsgBuffer; //!< buffer of output message
     double projectedArea;                       //!< [m^2] Area of the sensor projected along the sun vector.
     SpicePlanetStateMsgPayload sunData;         //!< [-] sun message input buffer
     SCStatesMsgPayload stateCurrent;            //!< [-] Current spacecraft state
     double shadowFactor;                        //!< [-] solar eclipse shadow factor from 0 (fully obscured) to 1 (fully visible)
-    double T;                                   //!< [C] Current temperature
+    double trueT;                               //!< [C] Current temperature
```

Comment:
:large_orange_diamond: A better name is trueTemp or trueTemperature. Likewise for sensedT.